articles/tour/bits-qubits.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Bits and quibits
+description: What is the difference between bits and quibits?
+author: MikeDodaro
+ms.author: Cathy.Palmer@microsoft.com
+uid: microsoft.quantum.tour.bits-quibits
+ms.date: 05/02/2019
+ms.topic: article
+---
+
+# Bits and quibits
```

Comment:
As above, I think this was accidently included from #412.

Reply:
Removed the article.